### PR TITLE
CMR-3935

### PR DESCRIPTION
**Pull request by zimzoom**

### CODE REVIEW

File: umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10.clj [hunk 1]
```diff
@@ -166,16 +166,21 @@
...
-                              (date/with-default-date (date/metadata-update-date c)))]))
+  (remove nil? (list
+    [:Metadata_Creation (f/unparse (f/formatters :date)
+                                   (date/with-default-date (date/metadata-create-date c)))]
+    [:Metadata_Last_Revision (f/unparse (f/formatters :date)
+                                        (date/with-default-date (date/metadata-update-date c)))]
+                (if (date/metadata-review-date c)
+                  [:Metadata_Future_Review (f/unparse (f/formatters :date)
+                                                      (date/metadata-review-date c))])
+                (if (date/metadata-review-date c)
```
Comment (eereiter) [inline umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10.clj:181]: This should be (if (date/metadata-delete-date c)

**Review by eereiter — CHANGES_REQUESTED** [review]

File: tea-config-generator/serverless.yml [hunk 2]
```diff
@@ -17,13 +17,14 @@ provider:
...
 
 package:
-#  individually: true
   exclude:
     - node_modules/**
 
 functions:
   capabilities:
     handler: capabilities.capabilities
+    description: Return a summery of the calls that can be made to this application
```
Comment (jaybarra) [inline tea-config-generator/serverless.yml:26]:
```suggestion
    description: Return a summary of the calls that can be made to this application
```

**Review by jaybarra — APPROVED** [review]

**Review by eereiter — APPROVED** [review]

File: tea-config-generator/serverless.yml (hunk 2 above)
Comment (jceaser) [reply tea-config-generator/serverless.yml:26]: I caused this.

**Review by jceaser — APPROVED** [review]
I would like to know what is up with the merge and why the tea stuff showed up here, but other then that I see nothing wrong.

**Pull request merged**